src/test/java/com/zilen/weather/service/WeatherServiceTest.java

```diff
@@ -15,8 +17,10 @@
     private RestTemplate restTemplate = Mockito.mock(RestTemplate.class);
     private String url = "url";
     private String appid = "appid";
+    @Autowired
+    private WeatherRepository weatherRepository;
```

Comment:
Autowired doesn't work for our tests. You don't raise any context for unit tests.